### ISSUE TITLE
is it possible to run openethereum on other chains such as bsc?

### ISSUE THREAD

**Issue by omryr** [openethereum/openethereum#573]
Hi so i was wondering if its possible to run the node with other chains? i couldnt find anything about it so far

**Comment by kamikazechaser** [comment]
Yes you can. You need to write your own chainspec. Something like this -> https://github.com/vporton/open-bsc/blob/main/ethcore/res/ethereum/bsc.json

Apparently the performance isn't good -> https://github.com/binance-chain/bsc/issues/377#issuecomment-905150381

**Comment by omryr** [comment]
is it usually about the same params for all chains? and if not where would i find these params?

**Comment by kaiuri** [comment]
> is it usually about the same params for all chains? and if not where would i find these params?

🙅🏻 Probably not, but I'm not completely sure, anyhow, you should not try copy/paste chainspecs

**Comment by q9f** [comment]
> 🙅🏻 Probably not, but I'm not completely sure, anyhow, you should not try copy/paste chainspecs

You totally can try that. Download the bsc.json from the link above and run 

```
openethereum --chain bsc.json
```
